app/data/models/exemption-order.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (sequelize, DataTypes) => {
+  const exemptionOrder = sequelize.define('exemption_order', {
+    id: {
+      autoIncrement: true,
+      autoIncrementIdentity: true,
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true
+    },
+    exemption_order: {
```

Comment:
I personally don't like naming a column the same as its table name - as it can lead to confusion - but we've done it quite a few times already, so go with what you have